src/workbench/Workbench.tsx

```diff
@@ -60,6 +61,16 @@ const Workbench = () => {
     }
   }, [selection, setSelectedFile, files]);
 
+  useEffect(() => {
+    if (flags.translate) {
```

Comment:
Is this adding it twice in development? I don't think we can meaningfully clean it up (it'll still have been loaded if we remove the element) but we could add an id to it and check whether it already existed.

Reply:
a753d1cd30908ddb693f8904adcc9a6449ed1270